src/epgsql_sock.erl

```diff
@@ -72,7 +79,14 @@
                 results = [],
                 batch = [],
                 sync_required,
-                txstatus}).
+                txstatus,
+                repl_last_received_lsn,
+                repl_last_flushed_lsn,
+                repl_last_applied_lsn,
+                repl_feedback_required,
+                repl_cbmodule,
+                repl_cbstate,
+                repl_receiver}).
```

Comment:
I don't know if it's a good idea or not, but let's discuss pros and cons...
What I think is that replication protocol unlikely will be most useful feature, but it adds lots of fields to state record. This means more heavy updates on this record (in terms of memory usage and garbage collection). Maybe it's better to use nested record for replication fields?
Like 
```
-record(repl_state, {
  last_received_lsn,
  last_flushed_lsn,
  last_applied_lsn,
  feedback_required,
  cbmodule,
  cbstate,
  receiver
}).

-record(state, {
  ...
  repl :: #repl_state{} | undefined
}).
```

@davidw what do you think?